packages/td-tools/src/thing-model-helpers.ts

```diff
@@ -138,7 +138,7 @@ export class ThingModelHelpers {
     public static getModelVersion(data: ThingModel): string | undefined {
         if (
             "version" in data &&
-            data.version &&
+            data.version != null &&
             typeof data.version === "object" &&
             "model" in data.version &&
             typeof data.version.model === "string"
```

Comment:
Maybe as a last-minute suggestion: I think these checks could be replaced with `typeof data?.version?.model === "string"` (or alternatively the whole method with `return data?.version?.model`).

Reply:
I see, thank you for making the adjustment! :)